src/EditorFeatures/CSharpTest2/Recommendations/RefKeywordRecommenderTests.cs

```diff
@@ -169,14 +169,6 @@ public async Task TestNotAfterOut()
     void Goo(out $$");
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        public async Task TestNotAfterThis()
-        {
-            await VerifyAbsenceAsync(
-@"static class C {
-    static void Goo(this $$");
-        }
```

Comment:
i would have preferred this not go away.  rather, the expected outcome could have just been updated.

Reply:
`TestExtensionMethods_FirstParameter_AfterThisKeyword` is the exact test you're talking about, with a different name

Reply:
Right.  It's just taht i would have preferred to just update the existing test versus removing. it makes it easier to tell without having to track that the test came back :)  It's not a big deal, just something small that helps :)

Reply:
> It's not a big deal, just something small that helps :)

Agreed. Wouldn't help now but does save a bit of time for cold reviews. It's most important for changes that impact a large number of tests, which clearly isn't the case here. 👍